### DISCUSSION TITLE
The language of the book is not supported.

### DISCUSSION THREAD

**Discussion by novabum** [xxyzz/WordDumb#70] [General]
### Describe the bug

Hi,
I am sorry, but I could not find any other way to contact you. I am a English-Hungarian translator and I would like to offer my help in adding Hungarian to WordDumb. If you are interested, hit me up.

Peter

### System Information

All systems.

### Error message

```text
The language of the book is not supported.
```

### Reproduce steps

1. You can contact me via kukacon (at) indamail.hu
Thank you

**Comment by xxyzz** [comment]
Hi @novabum, thanks for your interest in this project. You could upload the translation file in this discussion or create a pull request. Please read the document of adding translation file at [here](https://github.com/xxyzz/WordDumb/blob/master/docs/CONTRIBUTING.md#add-translations).

Update:
Do you mean the Hungarian books are not supported? I use [spaCy models](https://spacy.io/models) to find X-Ray entities in books, but spaCy doesn't have Hungarian models yet.

Maybe I'll add support of more languages for Word Wise once [wiktextract](https://github.com/tatuylonen/wiktextract) can parse non-English Wiktionary files.
